docs/howto/ayx-api-example-remove-all-users.md

```diff
@@ -0,0 +1,96 @@
+**Overview**
+
+This guide is for the use case of any workspace admin who needs to clear out a workspace except themselves. This is especially useful for universities using Alteryx Analytics Cloud when they have new classes of students in their workspace each semester.
+
+**Guide**
+---------
+
+This guide assumes limited knowledge of Python, but some research and debugging may be required by the user to set up Python and Pip Steps:
+
+1. Make sure you have Python installed by running both of the following in your terminal (whichever works, you'll use the same prefix for later commands):
+   1. `python --version`
+   2. `python3 --version`
```

Comment:
We can probably just use python commands, not python3 - Python 2 has been past EOL for like 4 years now, so short of some VERY ancient linux distros, we can probably safely assume `python` will symlink to `python3`